### PR TITLE
WMS-801 | Add "empty states" component

### PR DESCRIPTION
**Pull request by PedroHLC**
#### :thinking: What?

Implements this baby: https://www.figma.com/file/piyT6sDqja7yG7QarvmAZ9/Empty-States

#### :man_shrugging: Why?

* Good guy Sufyian noticed this pattern repeating a lot between screens and decided to upgrade it to a component.
* We're using this [in WMS Phase 4](https://www.figma.com/file/DTyoSveNksHpZCdU8a7BNs/Dashboard?node-id=1855%3A26417)

#### :pushpin: Jira Issue

[WMS-801](https://paacklogistics.atlassian.net/browse/WMS-801)

#### :no_good: Blocked by

* ~I need to add that "Fix" icon into paack-ui-assets.~ SOLVED

#### :clipboard: Pending

- [x] Documentation.
- [x] Add terms to POEditor.

### CODE REVIEW

**Comment by PedroHLC** [comment]
If someone wants to review it, the pending tasks don't involve coding and then won't affect reviewing...

File: src/UI/ContentPlaceholders.elm [hunk 1]
```diff
@@ -0,0 +1,236 @@
+module UI.ContentPlaceholders exposing
```
Comment (renanpvaz) [inline src/UI/ContentPlaceholders.elm:1]: I think the name should be in singular form like the rest of the components. The showcase module is fine though

File: src/UI/ContentPlaceholders.elm [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+
+-}
+custom :
+    { icon : String -> Icon
+    , title : String
+    , body : String
+    }
+    -> ContentPlaceholder
+custom prop =
+    Custom prop defaultOptions
```
Comment (renanpvaz) [inline src/UI/ContentPlaceholders.elm:92]: is it meant to support a custom placeholder? Will this be used anywhere?

Comment (PedroHLC) [reply src/UI/ContentPlaceholders.elm:92]: Yep, here: https://www.figma.com/file/DTyoSveNksHpZCdU8a7BNs/Dashboard?node-id=1855%3A26417

File: src/UI/ContentPlaceholders.elm [hunk 3]
```diff
@@ -0,0 +1,236 @@
...
+{-| One of the ready-to-use constructors.
+Used in impossible-states and empty unmanaged lists.
+
+    ContentPlaceholders.nothingToSeeHere
+        |> ContentPlaceholders.renderElement renderConfig
+
+-}
+nothingToSeeHere : ContentPlaceholder
+nothingToSeeHere =
+    NothingToSeeHere defaultOptions
```
Comment (jouderianjr) [inline src/UI/ContentPlaceholders.elm:104]: isn't easy to just return as a "custom" with the nothing to see here data set? what is the advantage of having a separated type for that?

File: src/UI/ContentPlaceholders.elm [hunk 4]
```diff
@@ -0,0 +1,236 @@
...
+
+
+defaultOptions : Options
+defaultOptions =
+    { size = Large }
+
+
+{-| With `withSize`, you'll be able to scale the icon between the [standard sizes][size].
```
Comment (jouderianjr) [inline src/UI/ContentPlaceholders.elm:112]: it's actually the icon size and the text size right?

File: src/UI/ContentPlaceholders.elm (hunk 3 above)
Comment (PedroHLC) [reply src/UI/ContentPlaceholders.elm:104]: No need to pass the locales to it at this point, just once at `renderElement`

File: src/UI/ContentPlaceholders.elm (hunk 4 above)
Comment (PedroHLC) [reply src/UI/ContentPlaceholders.elm:112]: Oops, my mistake

**Review by renanpvaz — APPROVED** [review]

**Review by kress95 — APPROVED** [review]

**Pull request merged**